scripts/generate_metadata.mjs

```diff
@@ -0,0 +1,52 @@
+import fs from "fs";
+import fetch from "node-fetch";
+
+const MIN_VERSION = "95.0.0"; // Update this when PR is about to be merged
+
+const fetchVersions = async () => {
+  const response = await fetch("https://registry.npmjs.com/carbon-react");
+  const json = await response.json();
+
+  return Object.keys(json.versions);
+};
+
+const formatVersions = (versions) => {
+  const filteredVersions = versions
+    .slice(versions.indexOf(MIN_VERSION))
+    .reverse();
+  const versionsJson = {
+    versions: {},
+  };
+
+  filteredVersions.forEach((item) => {
+    versionsJson.versions[
+      `v${item}`
+    ] = `https://carbon.sage.com/archive/v${item}/index.html`;
+  });
+
+  return JSON.stringify(versionsJson);
+};
+
+export const writeFile = (jsonString) => {
+  fs.mkdirSync("metadata", { recursive: true }, (err) => {
```

Comment:
Do you need the `recursive` flag?

Reply:
Ah I just had that in for testing when I was overwriting the previously created metadata directory, in the CI environment we won't need it so I'll take it out